examples/nlp/token_classification_infer.py

```diff
@@ -0,0 +1,114 @@
+import argparse
+import os
+
+import numpy as np
+from sklearn.metrics import classification_report
+
+import nemo
+import nemo_nlp
+from nemo_nlp import NemoBertTokenizer
+from nemo_nlp.utils.nlp_utils import get_vocab
+
+# Parsing arguments
+parser = argparse.ArgumentParser(description='NER with pretrained BERT')
+parser.add_argument("--max_seq_length", default=128, type=int)
+parser.add_argument("--fc_dropout", default=0, type=float)
+parser.add_argument("--pretrained_bert_model",
+                    default="bert-base-cased", type=str)
+parser.add_argument("--num_classes", default=9, type=int)
+parser.add_argument("--none_label", default='O', type=str)
+parser.add_argument("--queries", action='append',
+                    default=['we bought four shirts from the nvidia gear store\
+                    in santa clara', 'Nvidia is a company'],
+                    help="Example: --queries 'San Francisco' --queries 'LA'")
+parser.add_argument("--add_brackets", action='store_false',
+                    help="Whether to take predicted label in brackets or \
+                    just append to word in the output")
+parser.add_argument("--work_dir", default='output/checkpoints', type=str)
+parser.add_argument("--labels_dict", default='label_ids.csv', type=str)
+parser.add_argument("--amp_opt_level", default="O0",
+                    type=str, choices=["O0", "O1", "O2"])
+
+args = parser.parse_args()
+print(args)
```

Comment:
is it for debug? we can just copy the current file into the work_dir with files_to_copy=[__file__] in NeuralModuleFactory

Reply:
I'm removing it from here, but leaving nf.logger.info(args) in the training script, as it could be useful for debugging. Saving the current file won't store args specified in the command line.